internal/migrations/code/update_fingerprints_test.go

```diff
@@ -15,12 +15,18 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func newContextWithAccount(t *testing.T) context.Context {
+	return identity.WithTenant(t, context.Background())
+}
+
 func newContext(t *testing.T) context.Context {
 	return identity.WithTenant(t, context.Background())
```

Comment:
We have the context here as well, is that intentinal? I thought we wanted clean context for this one? :thinking:

Reply:
DAMN